firebase_admin/_user_mgt.py

```diff
@@ -726,6 +727,11 @@ def update_user(self, uid, display_name=None, email=None, phone_number=None,
                 custom_claims, dict) else custom_claims
             payload['customAttributes'] = _auth_utils.validate_custom_claims(json_claims)
 
+        if delete_providers is not None and isinstance(delete_providers, list):
```

Comment:
I think we want to validate the contents of the list too. #383 added a separate util method for this purpose:

```py
def validate_provider_ids(provider_ids, required=False):
    if provider_ids is None:
        if required:
            raise ValueError('Invalid provider IDs. The list must be non-empty.')
        else:
            return None
    for provider_id in provider_ids:
        validate_provider_id(provider_id, True)
    return provider_ids
```

Can we do something similar here?

Reply:
done.